sdk/docs/styles/DigitalAsset/Terms.yml

```diff
@@ -0,0 +1,24 @@
+# cspell: disable 
```

Comment:
add a note that there's a copy of this one in the Canton repo (link to the file location)

This way people could at least know.

Also create tech-debt issue to share the vale styles. One option I see is to add them to a public repository `digital-asset/vale-styles` and pull them in as part of the nix definition. Let's do that in a later step though.